test/CMakeLists.txt

```diff
@@ -388,12 +388,13 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_hfradar
 
 ecbuild_add_test( TARGET  test_${PROJECT_NAME}_argoclim
                   TYPE    SCRIPT
+                  ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
                   COMMAND ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
                   ARGS    netcdf
                           "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/argoClim2ioda.py
                           -i testinput/argoclim_test.nc
                           -o testrun/argoclim.nc
-                          -d 2019101600"
+                          -d 2004010200"
```

Comment:
looks like the actual output time was marked as :
"2004-02-16T00:00:00Z"
"2004-03-16T00:00:00Z"
which is different from the -d 2004010200

Reply:
yeah, I know. I'm not sure what we should do for timing for this data. it's different from other type of data we have. the complete data file (you can find it here https://sio-argo.ucsd.edu/RG_Climatology.html) contains the average monthly data from 20040101 to end of 2018. Here just a couple of points of data has been saved as a sample test file. I'm not sure how the time is going to be used for these kind of data.

Reply:
looks like so. Thank you for letting me know.

Reply:
sorry, one last doubt- should ObsError all be zero?

Reply:
Well, The obs error is set to 0 and qc to 1 by @aerorahul  2 years ago, and as I know this converter has not been used so far. Maybe he can answer better. It's not very clear to me how and when we are going to use this data. Probably we need to revisit this converter later whn we are going to actually use it.

Reply:
This is a converter of the Argo climatology.  It is not your typical observation dataset.  The intended purpose of this dataset was not for assimilation, but rather for providing a reference for comparison of the model and analysis departures from climatology.  

That being said, I am not sure what value would be appropriate for observation error, if this dataset was considered for assimilation.

Reply:
ARGO_TEMPERATURE_ANOMALY
seems to be the obs error,
if using ARGO_TEMPERATURE_MEAN + ARGO_TEMPERATURE_ANOMALY as ObsValue

Reply:
Agree with @lliu1220. Good point.
ARGO has a reported salinity drift in 2016 which was supposedly corrected in the climatology.